test/testcafe/spec/Identity_spec.js

```diff
@@ -45,4 +53,19 @@ test(`should have correct display texts`, async t => {
 
   const needhelpLinkText = identityPage.getNeedhelpLinkText();
   await t.expect(needhelpLinkText).eql('Need help signing in?');
+});
+
+test(`should show global error for invalid user`, async t => {
+  await t
+    .removeRequestHooks(mock)
+    .addRequestHooks(identifyLockedUserMock);
+
+  const identityPage = await setup(t);
+  await identityPage.fillIdentifierField('Test Identifier');
+
+  await identityPage.clickNextButton();
+
+  await identityPage.waitForErrorBox();
+
+  await t.expect(identityPage.getGlobalErrors()).contains('You do not have permission to perform the requested action.');
```

Comment:
try to use `.eql`

Reply:
`.contains` is broad. `.eql` is too restrictive for string comparison. Main goal is to check that we have the error or not.

Reply:
prefer restricts **if possible**.. 
IMHO, `contains` is kind suggesting bad practice cause you don't know if you're getting more unnecessary content.

Reply:
But the thing is i also don't want to fail the test if someone just prefix or suffix something in the error message in api. Test still fulfill its purpose of identifying the flow. I think in selenium they use contains at various places.

Reply:
> don't want to fail the test if someone just prefix or suffix something in the error message in api